Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -103,12 +134,22 @@ public void deleteAllPage() {
     dataBase.close();
   }
 
-  public void insertScriptedData(String url, String title, String content, int expectedTime, String imgUrl) {
-    SQLiteDatabase dataBase = getWritableDatabase();
-    String escapedTitle = StringEscapeUtils.escapeHtml4(title);
-    String escapedContent = StringEscapeUtils.escapeHtml4(content);
-    dataBase.execSQL("INSERT INTO PAGE (URL, TITLE, CONTENT, EXPECTEDTIME, IMGURL) VALUES ('" + url + "',\"" + escapedTitle + "\", \"" + escapedContent + "\", " + String.valueOf(expectedTime) + " , '" + imgUrl + "');");
+  /**
+   * check url duplication from database data.
+   * @param url
+   * @return url duplication result. duplication -> true, not duplication -> false
+   */
+  public boolean isDuplicationUrl(String url) {
```

Comment:
In the semantic view, I recommend to use `isDuplicatedUrl` instead of `Duplication`.